Client.xcodeproj/project.pbxproj

```diff
@@ -4626,7 +4628,7 @@
 				"$(SRCROOT)/Carthage/Build/iOS/Telemetry.framework",
 				"$(SRCROOT)/Carthage/Build/iOS/SwiftProtobuf.framework",
 				"$(SRCROOT)/Carthage/Build/iOS/Leanplum.framework",
-				"$(SRCROOT)/Carthage/Build/iOS/Glean.framework",
+				"$(SRCROOT)/Carthage/Build/iOS/Static/Glean.framework",
```

Comment:
Doubt this change is needed. We'll need to investigate where the binary shows up. When built locally with Carthage, the A-S lib goes into a `Static` folder. However when pulled as binary dependency, Carthage does not put it into that subfolder. Maybe we should file this a bug to Carthage as it is confusing.

Reply:
This is what i added to bootstrap.sh to handle the fallback case of when the binary isn't available and a local build is performed: https://github.com/mozilla-mobile/firefox-ios/commit/bf1a50d6c9e36dc961ccbed08c0e76194d2c81b8

Reply:
I saw that on the iOS14 branch, I can replicate it here if necessary.